example-projects/rdkafka-example/src/main.rs

```diff
@@ -0,0 +1,183 @@
+use clap::{App, Arg};
+use futures::StreamExt;
+use serde_json::json;
+use std::str::FromStr;
+use url::Url;
+
+use cloudevents::{EventBuilder, EventBuilderV10};
+use cloudevents_sdk_rdkafka::{BorrowedMessageExt, EventExt, FutureRecordExt};
+
+use rdkafka::client::ClientContext;
+use rdkafka::config::{ClientConfig, RDKafkaLogLevel};
+use rdkafka::consumer::stream_consumer::StreamConsumer;
+use rdkafka::consumer::{CommitMode, Consumer, ConsumerContext, Rebalance};
+use rdkafka::error::KafkaResult;
+use rdkafka::producer::{FutureProducer, FutureRecord};
+use rdkafka::topic_partition_list::TopicPartitionList;
+
+// run a Kafka lense or a Kafka Docker container to try out this example
+
+// A context can be used to change the behavior of producers and consumers by adding callbacks
+// that will be executed by librdkafka.
+// This particular context sets up custom callbacks to log rebalancing events.
+struct CustomContext;
```

Comment:
Remove this, you can use the default consumer context https://fede1024.github.io/rust-rdkafka/rdkafka/consumer/struct.DefaultConsumerContext.html